### PR TITLE
Gfonts edit11

### PR DESCRIPTION
**Pull request by asantewa**

### CODE REVIEW

File: ofl/candal/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1016/161139.344:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (felipesanches) [inline ofl/candal/debug.log:1]: what is this?

File: ofl/dosis/DESCRIPTION.en_us.html [hunk 2]
```diff
@@ -1,3 +1,8 @@
+<<<<<<< HEAD
```
Comment (felipesanches) [inline ofl/dosis/DESCRIPTION.en_us.html:1]: you've got a git conflict here.

File: ofl/frankruhllibre/debug.log [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[1016/171748.853:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (felipesanches) [inline ofl/frankruhllibre/debug.log:1]: again these log files. What's the purpose of these? It seems to me that they should not be included in the repo.

File: ofl/oswald/DESCRIPTION.en_us.html [hunk 4]
```diff
@@ -1,13 +1,16 @@
 <p>
-Oswald is a reworking of the classic style historically represented by the 'Alternate Gothic' sans serif typefaces.
-The characters of Oswald were initially re-drawn and reformed to better fit the pixel grid of standard digital screens.
-Oswald is designed to be used freely across the internet by web browsers on desktop computers, laptops and mobile devices.
+Oswald is a sans serif typeface that is synonymous to well-honed and acutely refined. 
+Every curve and angle of the letterform is finely adjusted to give it a visual balance that makes for pleasant reading. 
 </p>
 <p>
+<<<<<<< HEAD
```
Comment (felipesanches) [inline ofl/oswald/DESCRIPTION.en_us.html:6]: here's another git conflict

File: ofl/yanonekaffeesatz/DESCRIPTION.en_us.html [hunk 5]
```diff
@@ -1,6 +1,16 @@
+<<<<<<< HEAD
```
Comment (felipesanches) [inline ofl/yanonekaffeesatz/DESCRIPTION.en_us.html:1]: git conflict here as well

**Review by felipesanches — CHANGES_REQUESTED** [review]

**Comment by felipesanches** [comment]
@asantewa, If you need help fixing those git conflicts, let me know and I can offer some guidance.

**Comment by asantewa** [comment]
> @asantewa, If you need help fixing those git conflicts, let me know and I can offer some guidance.

thanks @felipesanches ! I have resolved them

**Pull request closed without merging**